### DISCUSSION TITLE
纯吐槽一下： sharetable.lua 里面的代码像一坨屎， 没打算重构一下的吗？

### DISCUSSION THREAD

**Discussion by aceyin** [cloudwu/skynet#1695] [General]
还有各种 service 的 lua 代码也是。

**Comment by Gowa2017** [comment]
恕我直言。你问这个问题。你比标题更像

**Comment by ylme** [comment]
作为程序员就说一下具体的问题，“像屎一样”的描述暴露了你的表达和逻辑不足，没有耐心。这样的程序员也不适合用 skynet 。

**Comment by hanxi** [comment]
不要抱怨，你可以这样 Fork -> rewrite -> pr -> rewrite -> pr ...

**Comment by terry8210** [comment]
找茬呢？

**Comment by terry8210** [comment]
哪里写的不错，如果改进，可以说的更详细哦。

**Comment by huahua132** [comment]
别🐕叫

**Comment by narutolai** [comment]
一共多少行，需要重构?，云风只是写了个例子
